sklearn/feature_extraction/text.py

```diff
@@ -1395,7 +1403,7 @@ def _make_int_array():
 
 
 class TfidfTransformer(TransformerMixin, BaseEstimator):
-    """Transform a count matrix to a normalized tf or tf-idf representation.
+    """Transform a count matrix to a normalized tf or tf-idf representation
```

Comment:
You need a final full stop here for sure. If it is too long you might ned to rephrase.